tests/unit/dataqualityfood.t

```diff
@@ -1885,4 +1885,20 @@ ok(
 	'Soluble fiber + Insoluble fiber exceeds total fiber'
 ) or diag explain $product_ref;
 
+# Test case: Mozzarella category with minimum number of ingredients
+my $product_ref = {
+	categories_tags => ['en:mozzarella'],
+	properties => {
+		minimum_number_of_ingredients => 3,
+	},
+	ingredients => ['cheese', 'salt', 'herbs'],
+	data_quality_warnings_tags => [],
+};
+
+ProductOpener::DataQualityFood::check_mozzarella_ingredients($product_ref);
+
+is(scalar @{$product_ref->{data_quality_warnings_tags}},
+	0, 'Product with Mozzarella category and sufficient ingredients should not raise a warning');
+
```

Comment:
Where is the test for when the warning is raised ?

Reply:
Let me go a head add back the check for when the warning is raised